test/Microsoft.TestPlatform.AcceptanceTests/TranslationLayerTests/CodeCoverageTests.cs

```diff
@@ -22,21 +22,21 @@ namespace Microsoft.TestPlatform.AcceptanceTests.TranslationLayerTests
     [TestCategory("Windows-Review")]
     public class CodeCoverageTests : CodeCoverageAcceptanceTestBase
     {
-        private IVsTestConsoleWrapper vstestConsoleWrapper;
+        private TestConsoleWrapperContext wrapperContext;
```

Comment:
Not sure about this name, context suggests something that we would insert into the class being used (the vstestconsole wrapper) to share some context with it. This seems more like a "container" or something. 

I don't see the additional properties of the `TestConsoleWrapperContext` used below. Maybe this field could remain IVSTestConsoleWrapper by grabbing the actual wrapper from the object, and all the additional changes in this class could be rolled back? (VSCode has nice "Revert changes in selected range" feature to do that when you right click selected text in a file diff).

Reply:
Agree